src/Microsoft.SqlTools.ManagedBatchParser/Localization/sr.cs

```diff
@@ -0,0 +1,6663 @@
+        }
+
+        public static string SchemaHierarchy_ColumnSetLabelWithTypeAndKeyString
+        {
+            get
+            {
+                return Keys.GetString(Keys.SchemaHierarchy_ColumnSetLabelWithTypeAndKeyString);
+            }
+        }
+
+        public static string UniqueIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.UniqueIndex_LabelPart);
+            }
+        }
+
+        public static string NonUniqueIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.NonUniqueIndex_LabelPart);
+            }
+        }
+
+        public static string ClusteredIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.ClusteredIndex_LabelPart);
+            }
+        }
+
+        public static string NonClusteredIndex_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.NonClusteredIndex_LabelPart);
+            }
+        }
+
+        public static string History_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.History_LabelPart);
+            }
+        }
+
+        public static string SystemVersioned_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.SystemVersioned_LabelPart);
+            }
+        }
+
+        public static string External_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.External_LabelPart);
+            }
+        }
+
+        public static string FileTable_LabelPart
+        {
+            get
+            {
+                return Keys.GetString(Keys.FileTable_LabelPart);
+            }
+        }
+
+        public static string DatabaseNotAccessible
+        {
+            get
+            {
+                return Keys.GetString(Keys.DatabaseNotAccessible);
+            }
+        }
+
+        public static string ScriptingParams_ConnectionString_Property_Invalid
+        {
+            get
+            {
+                return Keys.GetString(Keys.ScriptingParams_ConnectionString_Property_Invalid);
+            }
+        }
+
+        public static string ScriptingParams_FilePath_Property_Invalid
+        {
+            get
+            {
+                return Keys.GetString(Keys.ScriptingParams_FilePath_Property_Invalid);
+            }
+        }
+
+        public static string ScriptingListObjectsCompleteParams_ConnectionString_Property_Invalid
+        {
+            get
+            {
+                return Keys.GetString(Keys.ScriptingListObjectsCompleteParams_ConnectionString_Property_Invalid);
+            }
+        }
+
+        public static string StoredProcedureScriptParameterComment
+        {
+            get
+            {
+                return Keys.GetString(Keys.StoredProcedureScriptParameterComment);
+            }
+        }
+
+        public
```

Comment:
it looks like the resource files include strings not related to batch parser.  you should set up srgen to build localization resources only for the strings needed for this project, then remove those resources from the other project to avoid unneeded duplication.